.github/workflows/release.yml

```diff
@@ -41,30 +50,27 @@ jobs:
               echo "Release should build only from main branch"
               exit 1
           fi
-          if [ -z "$TRAVIS_API_TOKEN" ]; then
-              echo "TRAVIS_API_TOKEN is empty"
-              exit 1
-          fi
-
-          REQUEST_BUILD_BODY="
-          {
-            \"request\": {
-              \"branch\": \"$branch_name\",
-              \"sha\": \"$commit_sha\",
-              \"merge_mode\": \"deep_merge_append\",
-              \"config\": {
-                \"env\": {
-                  \"global\": [
-                    \"RELEASE_DATREE_PROD=true\",
-                    \"RELEASE_VERSION=${{ needs.define_version_job.outputs.version }}\"
-                  ]
-                }
-              }
-            }
-          }
-          "
-
-          REQUEST_BUILD_BODY=$REQUEST_BUILD_BODY ./.github/workflows/request-travis-build-and-wait-until-finished.sh
+      - name: Setup Go environment
+        uses: actions/setup-go@v2.1.4
+        with:
+          go-version: 1.18
+      - name: GitHub Config Identity
+        run: |
+          git config --global user.email "ci@datree.io"
+          git config --global user.name "datree-ci"
+      - name: Set AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Run deploy release candidate file
+        env:
+          RELEASE_VERSION: ${{ needs.define_version_job.outputs.version }}
+        run: |
+          chmod +x ./scripts/release.sh
```

Comment:
can we just run it with `bash ./scripts/release.sh`? instead of `chmod`.
I think we do (both here in your previous PR).
shorter and more readable